burn-import/README.md

```diff
@@ -39,7 +39,7 @@ Follow these steps to import an ONNX model into your Burn project:
 
    ```rust
    pub mod mnist {
-       include!(concat!(env!("OUT_DIR"), "/model/mnist.rs"));
+       include!(concat!(env!("OUT_DIR"), "/model/model_name.rs"));
    }
    ```
```

Comment:
Wrong module name!

Reply:
It should be the same as onnx file name.

Reply:
Ok, so can we leave as it is now?

Reply:
I would put model_name